src/scripts/add-user-to-group.ts

```diff
@@ -0,0 +1,121 @@
+import { Argv } from 'yargs';
+import { RootCommand } from '..';
+import { Response } from 'node-fetch';
+
+import { getUser } from './services/user-service';
+import { getGroup, userExistsInGroup } from './services/group-service';
+import { oktaManageClient, OktaConfiguration } from './services/client-service';
+
+const addUserToGroup = async (
+  oktaConfiguration: OktaConfiguration,
+  user: string,
+  group: string
+): Promise<Response> => {
+  const client = oktaManageClient(oktaConfiguration, ['groups', 'users']);
+
+  const maybeOktaUser = await getUser(user, client);
+  const maybeOktaGroup = await getGroup(group, client);
+
+  const throwOnMissing = (userMissing: boolean, groupMissing: boolean) => {
```

Comment:
Don't use flags like `groupMissing` to control behaviour in this way. This should be two functions.

Reply:
I constructed it that way so that if both a user and group did not exist, then you would receive an error message saying that both does not exist. Having separate commands would mean that it would have to be either or (throwing will terminate the imperative flow).

Reply:
@dspasojevic Should I write that in as a comment?

Reply:
There are better ways of achieving this outcome. Failing slow - as you have done here - is definitely the preferred practice. Doing it in this way is not ideal. For this PR please split this function apart, getting a slightly worse experience. I will take you through doing a similar thing in a scalable way next week.

Reply:
Separate function it will be then - implemented in next commit.